lib/babe/epoch_handler.go

```diff
@@ -23,6 +20,7 @@ var (
 )
 
 type epochHandler struct {
+	slotHandler *slotHandler
```

Comment:
nit why is this a pointer? I think we can make it a value for now (until we would need to use an interface eventually later)